src/krate.rs

```diff
@@ -550,22 +550,22 @@ pub fn summary(req: &mut Request) -> CargoResult<Response> {
                                         WHERE updated_at::timestamp(0) !=
                                               created_at::timestamp(0)
                                         ORDER BY updated_at DESC LIMIT 10"));
-    let most_downloaded = try!(tx.prepare("SELECT * FROM crates \
-                                           ORDER BY downloads DESC LIMIT 10"));
+    let most_directly_depended_upon = try!(tx.prepare("SELECT * FROM crates \
+                                        ORDER BY dependencies-on DESC LIMIT 10"));
```

Comment:
I can't see this column defined in the "schema" (as in, it's not mentioned in the [migration code](https://github.com/rust-lang/crates.io/blob/743bfc980b76c914d545b41d6f4cb601a5c1d213/src/bin/migrate.rs)), am I missing something?

(If it turns out this isn't defined, one may wish to look at `reverse_dependencies` which does essentially exactly the same thing.)